misk-zookeeper-testing/build.gradle

```diff
@@ -0,0 +1,15 @@
+buildscript {
+  dependencies {
+    classpath dep.kotlinNoArgPlugin
+  }
+}
+
+apply plugin: 'kotlin-jpa'
+
+dependencies {
+  compile dep.curatorFramework
+  compile dep.zookeeper
+  compile  dep.docker
```

Comment:
/nit extra space

Reply:
weird. intellij did this during the refactor. bad intellij.